catalogue/webapp/pages/download.tsx

```diff
@@ -27,29 +21,30 @@ import { GetServerSideProps, NextPage } from 'next';
 import { appError, AppErrorProps } from '@weco/common/services/app';
 import { getServerData } from '@weco/common/server-data';
 import { looksLikeCanonicalId } from 'services/catalogue';
-import { looksLikeSierraId } from '@weco/common/services/catalogue/sierra';
+import { fetchIIIFPresentationManifest } from '../services/iiif/fetch/manifest';
+import { transformManifest } from '../services/iiif/transformers/manifest';
 
 type Props = {
   workId: string;
-  sierraId: string;
-  manifest?: IIIFManifest;
-  work?: Work;
+  transformedManifest: TransformedManifest;
+  work: Work | undefined;
 };
 
 const DownloadPage: NextPage<Props> = ({
   workId,
-  sierraId,
-  manifest,
+  transformedManifest,
   work,
 }) => {
-  const title = (manifest && manifest.label) || (work && work.title) || '';
+  const { title, downloadEnabled, downloadOptions, iiifCredit } =
+    transformedManifest;
+  const displayTitle = title || work?.title || '';
   const iiifImageLocation = work
     ? getDigitalLocationOfType(work, 'iiif-image')
     : null;
   const iiifPresentationLocation = work
     ? getDigitalLocationOfType(work, 'iiif-presentation')
     : null;
-  const digitalLocation = iiifImageLocation || iiifPresentationLocation;
+  const digitalLocation = iiifImageLocation || iiifPresentationLocation; // TODO here we favour imageLocation over manifestLocation
```

Comment:
What's this TODO telling us? That in future we want to prefer manifest location? Or some other change?